kubernetes-pipeline/requirements.yaml

```diff
@@ -1,10 +1,10 @@
 dependencies:
 - name: spinnaker
-  version: 1.23.0
+  version: 2.2.0
```

Comment:
@Aaquiff in the PR https://github.com/wso2/kubernetes-pipeline/pull/86/files#diff-06237eb8818c51efa0628c6bea51a77bR68, the Halyard version needed to be explicitly set to gain Helm version 3 rendering support and since, the then stable release version `1.23.0` supported a lower version of Halyard, by default.

Shall we please avoid setting these values explicitly in the `values.yaml` since the latest versions are available by default in the new chart?

Reply:
Fixed.